### DISCUSSION TITLE
空洞里网太差了

### DISCUSSION THREAD

**Discussion by ADAD33** [share121/inter-knot#2268] [其他]
![微信截图_20240815211442](https://github.com/user-attachments/assets/66f5edc3-2504-4b20-aff5-ce8d7935c823)
刚刚登绳网，**等半天就等出这个**
有没有人改善一下空洞内的网络啊

**Comment by Kamisato100** [comment]
这边建议您使用H.D.D.系统实现空洞网络连接呢

**Comment by Genshin-Neuvillette** [comment]
你这空洞的名字是不是叫生物圈
